### ISSUE TITLE
thumbnail generation error

### ISSUE THREAD

**Issue by mshcheglov** [psidex/playlist2podcast#3]
Searching thumbnail url I got **KeyError**:
```
Traceback (most recent call last):
  File "/home/playlist2podcast/main.py", line 217, in <module>
    main()
  File "/home/playlist2podcast/main.py", line 212, in main
    p.update()
  File "/home/playlist2podcast/main.py", line 91, in update
    self.feedify(pod)
  File "/home/playlist2podcast/main.py", line 157, in feedify
    pixel_count = thumb["height"] * thumb["width"]
KeyError: 'height'
```

I thing this happens because something changed in Youtube API. Anyway **pod.playlist_meta["thumbnails"]** contains:
- {'url': 'https://...', 'height': 900, 'width': 900}
- {'url': 'https://...', 'id': 'avatar_uncropped', 'preference': 1}
- {'url': 'https://...', 'height': 175, 'width': 1060, 'preference': -10}
- {'url': 'https://...', 'height': 188, 'width': 1138, 'preference': -10}
- {'url': 'https://...', 'height': 283, 'width': 1707, 'preference': -10}
- {'url': 'https://...', 'height': 351, 'width': 2120, 'preference': -10}
- ...
- {'url': 'https://...', 'height': 1080, 'width': 1920, 'preference': -10}
- {'url': 'https://...', 'height': 1192, 'width': 2120, 'preference': -10}
- {'url': 'https://...', 'id': 'banner_uncropped', 'preference': -5}

As you can see not all thumbnails contain '**height**' and '**width**'.

**Comment by mshcheglov** [comment]
PR: https://github.com/psidex/playlist2podcast/pull/4

**Comment by psidex** [comment]
Thanks, looks good 👍 Have merged #4
